### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by vishrutshah**

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@vishrutshah__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <span>We will now validate the agreement and then real humans will evaluate your PR.</span>

TTYL, AZPRBOT;

File: README.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ gem 'azure_mgmt_network'
 ```
 
 Be aware the Azure Resource Manager Ruby SDK is in preview and will likely have breaking interface changes in upcoming
-releases.
+releases. An increase number in Minor version may indicate potential breaking changes.
```
Comment (veronicagg) [inline README.md]: How about: "An increased number in Minor version may indicate breaking changes."?

**Comment by veronicagg** [comment]
@vishrutshah Left a minor comment, rest looks good. Thanks!

File: README.md (hunk 1 above)
Comment (vishrutshah) [inline README.md]: Sounds better. Updated.

**Pull request merged**